projects/m5/002-polygon-area-calculator/js/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "jest": "^29.3.1"
+  },
+  "scripts": {
+    "test": "jest"
```

Comment:
Try: "test": "jest --coverage" ;-)

Reply:
Interesting! it shows the blocks that are not tested in the list of test I provided... In thist case I didn't test. In my case the method #validateInputNegativeN()